code/modules/mob/living/simple_animal/bees.dm

```diff
@@ -166,14 +198,37 @@
 		pixel_x = rand(-12,12)
 		pixel_y = rand(-12,12)
 
-	if(!parent && prob(10))
+	if(!parent && prob(8-strength))
 		strength -= 1
 		if(strength <= 0)
-			qdel(src)
-		else if(strength <= 5)
-			icon_state = "bees[strength]"
-
-	//debugging
-	/*icon_state = "[strength]"
-	if(strength > 5)
-		icon_state = "unknown"*/
+			death()
+		else
+			update_icons()
+
+
+/mob/living/simple_animal/bee/update_icons()
+	if(strength <= 5)
+		icon_state = "bees[strength]"
+	else
+		icon_state = "bees_swarm"
+
+//Kill it with fire!
+/mob/living/simple_animal/bee/adjustFireLoss(damage)
+	damage *= 2
+	..()
+
+
+//No more grabbing bee swarms
+/mob/living/simple_animal/bee/attempt_grab(var/mob/living/grabber)
+	world  << "Calling bee attemptgrab!"
```

Comment:
I usually sweep my files for "world <<" before pushing. I missed this because of the accidental extra space.

world spam in live is bad, certainly. But its something we should usually catch in update testing. 
IMO, log debug should really only be used for things we intend to leave in, for production debugging, like the event debug messages